attachments/utils.py

```diff
@@ -14,6 +14,12 @@
 import json
 import uuid
 
+def get_template_path():
+    try:
+        apps.get_app_config('bootstrap')
+    except:
+        return 'attachments/'
+    return 'attachments/bootstrap/'
```

Comment:
thinking about this, this is going to be a breaking change as written, can swap the files, so that bootstrap versions at attachments/
and non bootstrap versions are at attachments/base/ we can have a todo somewhere saying these will be swapped in the next major version